lib/symengine/matrix_base.rb

```diff
@@ -0,0 +1,7 @@
+module SymEngine
+  class MatrixBase
+    def inspect
+      "#<#{self.class}()>"
```

Comment:
Actually the sparse matrix is almost non-existent. The C++ code is still being developed. So, it's only possible to have an empty SparseMatirx, or have one, and then manually enter elements using `set`. This makes the `inspect` giving errors which comes from `__str__` method in C++ code.

That's why I have a more descriptive one for `DenseMatrix` separately.